### PR TITLE
fix: Attendee Form: Update field names

### PR DESCRIPTION
**Pull request by MeghalBisht**
#### Short description of what this resolves:
Reqd #6435

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by MeghalBisht** [comment]
@iamareebjamal is this correct? 😕

File: migrations/versions/rev-2021-01-20-23:37:40-a87d9f892893_.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# revision identifiers, used by Alembic.
+revision = 'a87d9f892893'
+down_revision = '9afc35c9f00f'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('ticket_holders', sa.Column('accept_receive_emails', existing_type=sa.Boolean(), nullable=True))
+    op.alter_column('ticket_holders', sa.Column('accept_share_details', existing_type=sa.Boolean(), nullable=True))
+    op.alter_column('ticket_holders', sa.Column('accept_video_recording', existing_type=sa.Boolean(), nullable=True))
```
Comment (iamareebjamal) [inline migrations/versions/rev-2021-01-20-23:37:40-a87d9f892893_.py:23]: What's changed? "Accept Video Recording" is still "Accept Video Recording" in the DB

**Comment by iamareebjamal** [comment]
https://github.com/fossasia/open-event-server/pull/7541/files

**Comment by MeghalBisht** [comment]
Okay, but it works?
koo

**Comment by iamareebjamal** [comment]
Only for new events, not for old

File: app/models/custom_form.py [hunk 2]
```diff
@@ -133,9 +133,9 @@
         "instagram": "Instagram",
         "gender": "Gender",
         "ageGroup": "Age Group",
-        "acceptVideoRecording": "Accept Video Recording",
-        "acceptShareDetails": "Accept Share Details",
-        "acceptReceiveEmails": "Accept Receive Emails",
+        "acceptVideoRecording": "Photo & video & text consent",
```
Comment (iamareebjamal) [inline app/models/custom_form.py:136]: Please make sure that this is actually what is needed, or else another migration will be needed

Comment (MeghalBisht) [reply app/models/custom_form.py:136]: In the issues @mariobehling mentioned these only

Comment (mariobehling) [reply app/models/custom_form.py:136]: Yes, I clarified that now. Actually sometimes is differs for events, but this seems to be suitable for the meantime until we can have the tick box for custom questions.

**Pull request merged**